content/python/encyclopedia/user-input-in-python.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: "User Input in Python" 
+Subjects:
+  - "Code Foundations"
```

Comment:
Let's take out `"Code Foundations"` and let's add `"Data Science"` or `"Web Development"`.